src/app/cube/home/home.component.ts

```diff
@@ -59,7 +59,7 @@ export class HomeComponent implements OnInit {
   ngOnInit() {
     this.statsService.getLearningObjectStats().then(stats => {
       this.usageStats.objects.released = stats.released;
-      this.usageStats.objects.underReview = stats.total - stats.released;
+      this.usageStats.objects.underReview = stats.total - stats.released - stats.working;
```

Comment:
Is "working" in reference to all objects that are unreleased?

Reply:
This is now changed so that "underReview" is set to stats.review, which is all of the objects under review.